website/content/en/docs/kubebuilder/operator-scope.md

```diff
@@ -0,0 +1,299 @@
+responsible for creating the permissions which allow the operator to have access to the resources.
+
+**NOTE** Only `ClusterRole` and `ClusterRoleBinding` manifests require changes to achieve this goal. 
+Ignore `<kind>_editor_role.yaml`, `<kind>_viewer_role.yaml`, and files a name pattern of `auth_proxy_*.yaml`.
+
+### Changing the permissions 
+
+To change the scope of the RBAC permissions from cluster-wide to a specific namespace, you will need to use `Role`s 
+and `RoleBinding`s instead of `ClusterRole`s and `ClusterRoleBinding`s, respectively.
+ 
+[`RBAC markers`][rbac-markers] defined in the controller (e.g `controllers/memcached_controller.go`)
+are used to generate the operator's [RBAC ClusterRole][rbac-clusterrole] (e.g `config/rbac/role.yaml`). The default
+ markers don't specify a `namespace` property and will result in a ClusterRole.
+
+Update the RBAC markers to specify a `namespace` property so that `config/rbac/role.yaml` is generated as a `Role`
+ instead of a `ClusterRole`.
+
+Replace:
+
+```go
+// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=cache.example.com,resources=memcacheds/status,verbs=get;update;patch
+```
+
+With namespaced markers:
+
+```go
+// +kubebuilder:rbac:groups=cache.example.com,namespace="my-namespace",resources=memcacheds,verbs=get;list;watch;create;update;patch;delete
+// +kubebuilder:rbac:groups=cache.example.com,namespace="my-namespace",resources=memcacheds/status,verbs=get;update;patch
+```
+
+And then, run `make manifests` to update `config/rbac/role.yaml`:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+...
+```
+
+We also need to update our `ClusterRoleBindings` to `RoleBindings` since they are not regenerated: 
+
+```yaml
+  
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: manager-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: manager-role
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: system
+```
+
+<!-- todo(camilamacedo86): The need for the RoleBinding show an issue tracked 
+in https://github.com/kubernetes-sigs/kubebuilder/issues/1496. --> 
+
+## Using environment variables for Namespace 
+
+Instead of having any Namespaces hard-coded in the `main.go` file a good practice is to use environment 
+variables to allow the restrictive configurations
+
+### Configuring Namespace scoped operators
+
+- Add a helper function in the `main.go` file:
+
+```go
+// getWatchNamespace returns the Namespace the operator should be watching for changes
+func getWatchNamespace() (string, error) {
+    // WatchNamespaceEnvVar is the constant for env variable WATCH_NAMESPACE
+    // which is the Namespace where the watch activity happens.
+    // this value is empty if the operator is running with clusterScope.
```

Comment:
I realize this is straight from our util but `watch activity happens` is awkward phrasing.
```suggestion
    // which specifies the Namespace to watch.
    // An empty value means the operator is running with cluster scope.
```